checkbox-provider/snap/hooks/configure

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# Copyright 2017-2022 Canonical Ltd.
+# All rights reserved.
+
+# this  script will be executed on the host at  snap installation
+# (not refresh)
+
+"""
+This hook reads `config_vars` file from the root of a snap and reenters this
+configuration using snapd's config facilities and rewrites the file to a
+checkbox config format in `$SNAP_DATA/checkbox.conf`.
+
+config_vars should list all the configuration variables in a `key=value`
+syntax. The line can list variable name only, if the variable should not have a
+default value. All keys should comprise of CAPS, numbers and undescores (_).
+E.g.:
+
+OPEN_N_SSID
+OPEN_BG_SSID
+STRESS_S3_WAIT_DELAY=60
+
+To change those values use the configure launcher.
+E.g.
+    $ checkbox-qat-classic.configure OPEN_N_SSID=my-wifi
+
+If you __really__ need to change those values using `snap set` command,
+you need to change CAPS to lowercase and underscores to dashes (-).
+
+E.g.
+    $ snap set checkbox-qat-classic open-n-ssid=my-wifi
+"""
+
+import os
+import sys
+
+sys.path.append(os.path.expandvars("$SNAP/usr/lib/python3/dist-packages"))
+sitepkgpath = "$SNAP/lib/python3.10/site-packages"
```

Comment:
I see version 3.10 included in the path here but version 3.12 is referenced in the `snapcraft.yaml`.

Reply:
Nice catch, thanks, fixed